### ISSUE TITLE
risky-file-permissions rule ignores FQCN

### ISSUE THREAD

**Issue by dessert1** [ansible-community/ansible-lint#1528]
##### Summary
The risky-file-permissions rule is triggered for the `copy` module, but not for `ansible.builtin.copy`.

##### Issue Type
- Bug Report

##### Ansible and Ansible Lint details
```console
$ ansible --version
ansible 2.10.7
  config file = /home/me/.ansible.cfg
  configured module search path = ['/home/me/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.9/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.9.2 (default, Feb 20 2021, 18:40:11) [GCC 10.2.0]
$ ansible-lint --version
ansible-lint 5.0.7 using ansible 2.10.7
```

- ansible installation method: OS package
- ansible-lint installation method: OS package

##### OS / ENVIRONMENT
Arch Linux

##### STEPS TO REPRODUCE
A playbook using `ansible.builtin.copy` without `mode` gives no violations, shortening the module name to just `copy` throws the risky-file-permissions warning as expected.
```console
$ cat site.yml
- hosts: all
  tasks:
    - name: copy that
      ansible.builtin.copy:
$ ansible-lint -pqq site.yml
$ sed -i 's/ansible\.builtin\.//' site.yml
$ ansible-lint -pqq site.yml
site.yml:3: risky-file-permissions
```

##### Desired Behaviour
`ansible-lint` should treat the `ansible.builtin.copy` module exactly like it does `copy`.

##### Actual Behaviour
`ansible-lint` doesn’t seem to work with FQCNs at least in this particular case. There may be other examples where modules are ignored because they’re called with their FCQN.

**Comment by oldATWiz** [comment]
I see this as well with ansible.builtin.command. Adding the FQCN removes the 'no-changed-when: Commands should not change things if nothing needs doing' violation.

```
ansible-lint --version
ansible-lint 5.0.9 using ansible 2.11.0
```
